src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubTriggerInputBindingStrategy.cs

```diff
@@ -54,7 +42,26 @@ public EventData[] BindMultiple(EventHubTriggerInput value, ValueBindingContext
             return value.Events;
         }
 
-        // GEt runtime instance of binding contract 
+        // Get the static binding contract
+        //  - gets augmented 
+        public Dictionary<string, Type> GetStaticBindingContract()
+        {
+            Dictionary<string, Type> contract = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
+            contract.Add("PartitionContext", typeof(PartitionContext));
+
+            // TODO: can we determine here if the binding is single dispatch?
```

Comment:
that's a good point - I'm not sure if this is possible today. The single strategy object handles both, and the caller is determining whether to call BindSingle or BindMultiple. Can you file a bug to me to followup on that?